modules/network-observability-enriched-flows.adoc

```diff
@@ -37,11 +38,29 @@ spec:
         targetHost: "ipfix-collector.ipfix.svc.cluster.local"
         targetPort: 4739
         transport: tcp or udp           <4>
-
-
+ -  type: OpenTelemetry                 <3>
+      openTelemetry:
+        targetHost: my-otelcol-collector-headless.otlp.svc
+        targetPort: 4317
+        type: grpc                      <5>
+        logs:                           <6>
+          enable: true
+        metrics:                        <7>
+          enable: true
+          prefix: netobserv
+          pushTimeInterval: 20s         <8>
+          expiryTime: 2m
+   #    fieldsMapping:                  <9>
+   #      input: SrcAddr
+   #      output: source.address
```

Comment:
Just checking, is this meant to be commented out in the YAML? Because it's showing up in the preview, just as commented out parameters

Reply:
Most of the users **should not** apply the fieldsMapping. This is just an example of remapping for the few users concerned.

That's why we decided to comment it in the example. It will avoid support in case users copy/paste the example without fully reading the context.

Reply:
Sorry, I'm delayed in responding to peer review feedback but I just wanted to say yep what Julien says. And the callout indicates its optional. Its mostly just for awareness of the feature, but its not necessarily something a user might need or want to do.